opm/simulators/aquifers/AquiferFetkovich.hpp

```diff
@@ -23,6 +23,11 @@ along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 
 #include <opm/simulators/aquifers/AquiferInterface.hpp>
 
+#include <opm/output/data/Aquifer.hpp>
```

Comment:
This is really nitpicking. I think it is sufficient to include it from the base class. 

It can be a style thing. You do not have to address this one. 

And similar applies to the two header files below. I think we only need `stdexcept`.

Reply:
> This is really nitpicking. I think it is sufficient to include it from the base class.
> 
> It can be a style thing. You do not have to address this one.

It is true, we do get that transitively from `AquiferInterface.hpp`.  On the other hand, I've recently come to prefer "include what you use" and the new implementation definitely uses/refers to  members of the structures declared in `opm/output/data/Aquifer.hpp`.